lib/js/components/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+	<div class="sectionHeader" :class="{ '-expandable': expandable, [sizeClass]: true }">
+		<div class="sectionHeader__textWrapper" @click="onTextWrapperClicked">
+			<ds-icon
+				v-if="iconLeft"
+				class="sectionHeader__iconLeft"
+				:icon="iconLeft"
+				:size="ICON_SIZES.SMALL"
+			/>
+			<span class="sectionHeader__text">{{ text }}</span>
+			<ds-icon
+				v-if="expandable"
+				class="sectionHeader__chevron"
+				:icon="ICONS.FA_CHEVRON_DOWN"
+				:rotation="chevronRotation"
```

Comment:
in `CardExpandable` we use `flipped-vertical` for expander. Should we unify animation in these components?

Reply:
`flipped-vertical` is perfect. I missed it, thanks.

Reply:
I changed `CardExpandable` to use `rotation` instead, per https://bethinkteam.slack.com/archives/C012R8RM3A4/p1651744363569849?thread_ts=1651659634.190469&cid=C012R8RM3A4